packages/ia-components/sandbox/IAReactComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+const debug = require('debug')('dweb-archive:ConfigDetailsComponent');
```

Comment:
Suggestion: Add comment around `debug` component - and how to configure.  You mentioned that it is a current node process but there is a client side workaround to ensure that it works.